spec/models/charge_spec.rb

```diff
@@ -1,10 +1,17 @@
 require 'rails_helper'
 
-RSpec.describe Charge, type: :model do
+describe Charge do
   it "has a valid factory" do
     FactoryGirl.create(:charge).should be_valid
   end
-  # it "is invalid without a booking id" do
-  #   FactoryGirl.create(:charge, booking_id: nil).should_not be_valid
-  # end
+
+  describe 'scope' do
+    it 'returns bondable charges with bond amount greater than 0' do
+      FactoryGirl.create(:charge, bond_amount: nil)
+      FactoryGirl.create(:charge, bond_amount: 0)
+      FactoryGirl.create(:charge, bond_amount: 100)
+
+      expect(Charge.bondable.count).to eq(1)
```

Comment:
This only checks the number of returned records. It doesn't check that the correct one is returned.